pkg/controller/datavolume-controller.go

```diff
@@ -334,6 +334,24 @@ func (c *DataVolumeController) updateCloneStatusPhase(pvc *corev1.PersistentVolu
 	}
 }
 
+func (c *DataVolumeController) updateUploadStatusPhase(pvc *corev1.PersistentVolumeClaim, dataVolumeCopy *cdiv1.DataVolume) {
+	phase, ok := pvc.Annotations[AnnPodPhase]
+	if ok {
+		switch phase {
+		case string(corev1.PodPending):
+			// TODO: Use a more generic Scheduled, like maybe TransferScheduled.
+			dataVolumeCopy.Status.Phase = cdiv1.UploadScheduled
+		case string(corev1.PodRunning):
+			// TODO: Use a more generic In Progess, like maybe TransferInProgress.
+			dataVolumeCopy.Status.Phase = cdiv1.UploadInProgress
```

Comment:
This is a bit misleading.  UploadInProgress makes me think someone has started sending data.  How about UploadReady?

Reply:
take into account that when the image is uploaded for real, the pod status is still running, and the status you will have in the datavolume is still UplaodReady...  There is not difference in the pods status (running) when the image is ready to be uploaded and when it's loaded...

Reply:
I prefer UploadReady as well.